inputs/manual_input.csv

```diff
@@ -0,0 +1,170 @@
+methane storage tank incl. compressor,FOM,2014,1.9,%/year,2014,"Guesstimate, based on hydrogen storage tank by DEA.",Based on assumptions for hydrogen storage tank incl. Compressor (by DEA).
+methane storage tank incl. compressor,lifetime,2014,30,years,2014,"Guesstimate, based on hydrogen storage tank by DEA.",Based on assumptions for hydrogen storage tank incl. Compressor (by DEA).
+LOHC loaded DBT storage,investment,2012,140.659340659341,EUR/t,2012,"Density via Wissenschaftliche Dienste des Deutschen Bundestages 2020, https://www.bundestag.de/resource/blob/816048/454e182d5956d45a664da9eb85486f76/WD-8-058-20-pdf-data.pdf , pg. 11.","Based on storage “General liquid hydrocarbon storage (crude)”, as similar properties are shared. Density of loaded LOHC H18-DBT is 0.91 t/m^3."
+LOHC loaded DBT storage,FOM,2012,6.25,%/year,2012,,"Based on storage “General liquid hydrocarbon storage (crude)”, as similar properties are shared."
+LOHC loaded DBT storage,lifetime,2012,30,years,2012,,"Based on storage “General liquid hydrocarbon storage (crude)”, as similar properties are shared."
+LOHC unloaded DBT storage,investment,2012,124.634858812074,EUR/t,2012,"Density via Wissenschaftliche Dienste des Deutschen Bundestages 2020, https://www.bundestag.de/resource/blob/816048/454e182d5956d45a664da9eb85486f76/WD-8-058-20-pdf-data.pdf , pg. 11.","Based on storage “General liquid hydrocarbon storage (crude)”, as similar properties are shared. Density of loaded LOHC H18-DBT is 0.91 t/m^3, density of unloaded LOHC H0-DBT is 1.04 t/m^3 but unloading is only to 90% (depth-of-discharge), assume density via linearisation of 1.027 t/m^3."
+LOHC unloaded DBT storage,FOM,2012,6.25,%/year,2012,,"Based on storage “General liquid hydrocarbon storage (crude)”, as similar properties are shared."
+LOHC unloaded DBT storage,lifetime,2012,30,years,2012,,"Based on storage “General liquid hydrocarbon storage (crude)”, as similar properties are shared."
+FT fuel transport ship,investment,2035,35000000,EUR,2020,"Assume comparable tanker as for LOHC transport above, c.f. Runge et al 2020, Table 10, https://papers.ssrn.com/abstract=3623514 .",
+FT fuel transport ship,capacity,2035,75000,t_FTfuel,2020,"Assume comparable tanker as for LOHC transport above, c.f. Runge et al 2020, Table 10, https://papers.ssrn.com/abstract=3623514 .",
+FT fuel transport ship,lifetime,2035,15,years,2020,"Assume comparable tanker as for LOHC transport above, c.f. Runge et al 2020, Table 10, https://papers.ssrn.com/abstract=3623514 .",
+FT fuel transport ship,FOM,2035,5,%/year,2020,"Assume comparable tanker as for LOHC transport above, c.f. Runge et al 2020, Table 10, https://papers.ssrn.com/abstract=3623514 .",
+Fischer-Tropsch synthesis,investment,2020,2100000,EUR/MW_FTfuel,2015,"Danish Energy Agency,  data_sheets_for_renewable_fuels_-_0003.xlsx , 102 Hydrogen to Jet .",
```

Comment:
I am working on automating FT, methanolisation, industrial heat pump and H2 pipeline in the `compile_cost_assumptions` rule. Now I found a unit discrepancy. Here, the unit is "EUR/MW_FTfuel", but the referenced sheet has as unit "M€ /MW Liquids/year" (it's the per year I am wondering about). Since the old values align much better with the first-mentioned unit, I am assuming that this is a mistake in the DEA database. Was that also your impression @euronion?

Reply:
Ah, yes see #29